docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_types.go

```diff
@@ -165,20 +165,28 @@ type CronJobStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 
-// CronJob is the Schema for the cronjobs API.
+// CronJob is the Schema for the cronjobs API
 type CronJob struct {
 	/*
 	 */
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+
+	// metadata is a standard object metadata
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
```suggestion
	metav1.ObjectMeta `json:"metadata,omitempty,omitzero"`
```

Reply:
I think it is a better option than pointer 
I was thinking and the pointer might complicated things